autoload/lsp/client.vim

```diff
@@ -109,7 +109,9 @@ function! s:on_stdout(id, data, event) abort
                     endtry
                     unlet l:ctx['on_notifications'][l:response['id']]
                 endif
-                unlet l:ctx['requests'][l:response['id']]
+                if has_key(l:ctx['requests'], l:response['id'])
+                    unlet l:ctx['requests'][l:response['id']]
```

Comment:
let us add a log if response id doesn't exist

Reply:
Done. If you prefer echo I can change it.